src/libs/ajax.js

```diff
@@ -1388,7 +1388,7 @@ const Apps = signal => ({
             name: diskName,
             size: diskSize,
             labels: {
-              saturnApplication: 'galaxy',
+              saturnApplication: appType,
```

Comment:
Hmm.. I don't think this is a valid change.

There are two things at play here. One is the backend appType, defined here (the all caps version) https://github.com/DataBiosphere/leonardo/blob/develop/core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala#L304

One is the frontend label, used for filtering in listApps calls, here: https://github.com/DataBiosphere/terra-ui/blob/dev/src/pages/workspaces/workspace/WorkspaceContainer.js#L182

In other words, this value needs to remain the lowercase 'galaxy' (the `galaxy.label`) for the UI to properly function, as the list calls rely on existing disks with this label.

Reply:
Interestingly, this has been working with the cases different. It appears that the backend call is case-insensitive because I verified that the following 2 things return the same thing:

```
Ajax(signal).Disks.list({ googleProject, creator: getUser().email, saturnApplication: "galaxy" })

Ajax(signal).Disks.list({ googleProject, creator: getUser().email, saturnApplication: "GALAXY" })
```
I will make the front-end code consistent though, using the appType since that is what we have available in ajax.js.

Reply:
that's good to know!